deployments/reviewer/libero-reviewer--prod.yaml

```diff
@@ -92,7 +92,7 @@ spec:
     browsertests:
       runPostRelease: false
       testFixture: Minimal
-      baseurl: libero-reviewer--prod-client-canary.prod
+      baseurl: "https://libero-reviewer--staging.elifesciences.org"
```

Comment:
Should that not be prod rather than staging?

Reply:
Woops, have changed this